src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Tables/Resource.sql

```diff
@@ -15,6 +15,7 @@
 )
 
 ALTER TABLE dbo.Resource SET ( LOCK_ESCALATION = AUTO )
+ALTER TABLE dbo.Resource ADD CONSTRAINT RawResourceLength CHECK (DATALENGTH(RawResource) > 49)
```

Comment:
Can we add this constraint as part of CREAT TABLE statement itself as this sql used for initialization and not for migration?

Reply:
I tried adding the constraint inside the create table, but was getting syntax errors.

Reply:
I tried and it works
![image](https://user-images.githubusercontent.com/57157506/165619401-a2d2c2fe-32f3-43f0-af4c-0a6cad22e7b8.png)

Reply:
Thanks for the update, I have changed it to be part of the CREATE